### PR TITLE
Linguisticprobing

### PR DESCRIPTION
**Pull request by sileod**
Adding Linguistic probing datasets from
What you can cram into a single $&!#* vector: Probing sentence embeddings for linguistic properties
 https://www.aclweb.org/anthology/P18-1198/

### CODE REVIEW

File: datasets/linguisticprobing/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  top_constituents:
+  - 10K<n<100K
+  tree_depth:
+  - 100K<n<1M
+  word_content:
+  - 100K<n<1M
+source_datasets: []
+task_categories:
+- text-classification
+task_ids: []
```
Comment (lhoestq) [inline datasets/linguisticprobing/README.md:32]: Looks like the dataset card is missing the content summary as well as the section titles.
More info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

You can leave the section contents with `[More Information Needed]`

**Review by lhoestq — COMMENTED** [review]
Looking good so far thanks !

Could you add the missing sections in the dataset card ? you can just have the titles and leave the content with `[More Information Needed]`

also it looks like there are also changes about the `discovery` dataset, could you include in this PR only changes for the `linguisticprobing` dataset and add `discoevry` in a separate one please ?

File: datasets/linguisticprobing/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  sentence_length:
+  - text-classification-other-sentence length prediction
+  subj_number:
+  - text-classification-other-subjects number prediction
+  top_constituents:
+  - text-classification-other-constituents prediction
+  tree_depth:
+  - text-classification-other-syntactic tree depth prediction
+  word_content:
+  - text-classification-other-words prediction
```
Comment (lhoestq) [inline datasets/linguisticprobing/README.md:60]: thanks for adding all these tags !

Can you also add the other sections of the dataset card ?
You can leave their content with `[More Information Needed]`
You can see the template here : https://github.com/huggingface/datasets/blob/master/templates/README.md

**Review by lhoestq — COMMENTED** [review]
Really cool thanks :) 

Before we merge, can you also add the other sections of the dataset card ?

Also can you try to reduce the sizes of the dummy_data.zip files (they're currently 17KB each).
To do so feel free to take a look inside the dummy_data.zip file for the `bigram_shift` config for example and remove all the unnecessary files about the other configs.

**Review by lhoestq — COMMENTED** [review]
Thanks ! did you get a chance to take a look at the dummy data files to try to reduce their sizes ?

**Comment by albertvillanova** [comment]
Thanks for your contribution, @sileod.

We are removing the dataset scripts from this GitHub repo and moving them to the Hugging Face Hub: https://huggingface.co/datasets

As you already created this dataset under your organization namespace (https://huggingface.co/datasets/metaeval/linguisticprobing), I think we can safely close this PR.

We would suggest you add a dataset card with the YAML tags, to make it searchable and discoverable.

Please, feel free to tell us if you need some help.

**Pull request closed without merging**